index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <!-- fontawesme link -->
+    <script src="https://kit.fontawesome.com/fc7e7ce012.js" crossorigin="anonymous"></script>
+    <!-- style css -->
+    <link rel="stylesheet" href="style.css">
+
+
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="navbar">
+                <ul class="top-navbar-menu">
+                    <li><a href="">Facebook</a></li>
+                    <li><a href="">Twitter</a></li>
+                    <li><a href="">English</a></li>
+                    <li><a href="">My Page</a></li>
+                    <li><a href="">Log out</a></li>
+                </ul>
+                <div class="bottom-navbar">
+                    <a class="logo" href="index.html">Logo</a>
+                    <ul class="bottom-navbar-menu">
+                        <li><a href="about.html">About</a></li>
+                        <li><a href="">Program</a></li>
+                        <li><a href="">Join</a></li>
+                        <li><a href="">Sponsor</a></li>
+                        <li><a href="">News</a></li>
+                        <li><a href="">Campiagn</a></li>
+                    </ul>   
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="info">
+            <div class="container">
+                <button class="nav-btn"><i class="fa-solid fa-bars fa-2x"></i></button>
```

Comment:
- [ ] This button I believe is the menu button and based on page semantics it should be placed inside your navbar, in a `nav` tag. Kindly re-position this button following the correct semantics.